lib/galaxy/webapps/galaxy/buildapp.py

```diff
@@ -256,6 +256,7 @@ def populate_api_routes( webapp, app ):
     webapp.mapper.resource( 'remote_file', 'remote_files', path_prefix='/api' )
     webapp.mapper.resource( 'group', 'groups', path_prefix='/api' )
     webapp.mapper.resource_with_deleted( 'quota', 'quotas', path_prefix='/api' )
+    webapp.mapper.connect( '/api/interactive_environment_ready', controller='interactive_environment_ready' )
```

Comment:
I know they're launched *from* dataset visualizations, but are they actually tied to them?  Since they ultimately end up being a history item, it feels like history/contents/<whatever_it_is> should show status.